vrk-kpa/suomidigi
# SUOPA-788 Article teaser

To test: 
- `make fresh drush-cr`
- Open mobile proto: https://xd.adobe.com/view/1067743e-f959-428d-664c-974d183bc243-6656/ and  desktop proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/
- Go to front page and look for "Viimeisimmät kehittämisesimerkit"
- Compare the new theme on article teasers and protos (above), make sure they're alike
